src/test/java/it/gov/pagopa/payment/service/qrcode/QRCodePreAuthServiceImplTest.java

```diff
@@ -25,20 +25,23 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.http.HttpStatus;
 
+import java.util.List;
+
 @ExtendWith(MockitoExtension.class)
 class QRCodePreAuthServiceImplTest {
 
   @Mock private TransactionInProgressRepository transactionInProgressRepository;
   @Mock private RewardCalculatorConnector rewardCalculatorConnector;
-
+  @Mock private AuditUtilities auditUtilitiesMock;
   private QRCodePreAuthService qrCodePreAuthService;
```

Comment:
let's keep separated blocks of not related parameters
```suggestion
  @Mock private AuditUtilities auditUtilitiesMock;
  
  private QRCodePreAuthService qrCodePreAuthService;
```